src/io/perun/core.clj

```diff
@@ -11,15 +12,15 @@
   "Return metadata on files. Files metadata is a list.
    Internally it's stored as a map indexed by `:path`"
   [fileset]
-  (-> fileset meta +meta-key+ vals))
+  (keep +meta-key+ (vals (:tree fileset))))
```

Comment:
Perhaps I should explain - `get-meta` returns data in the same shape as before - a seq of metadata maps.  This PR doesn't have any API changes, aside from not needing to deal with removed markdown files.